packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,37 @@
+# TextDecorationControl
+
+**Notice: This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.**
+
+![TextDecorationControl Element in Inspector Control](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/text-decoration-component.png?raw=true)
+
+
+## Usage
+
+```jsx
+import { __experimentalTextDecorationControl } from '@wordpress/block-editor';
+```
+
+Then, you can use the component in your block editor UI:
+
+```jsx
+<__experimentalTextDecorationControl
```

Comment:
```suggestion
<TextDecorationControl
```
If renaming the experimental component as suggested above, we'll also need to update the example here.